onos-classic/templates/onos-config-loader-script.yaml

```diff
@@ -38,6 +38,8 @@ data:
     NETCFG="/opt/configs/netcfg.json"
     RECONCILE_MODE="{{ .Values.config.reconcile_mode }}";
 
+    # run in tmp instead of root
+    cd /tmp
```

Comment:
Hi Randy, thanks for the patch! is this change to /tmp because of security concerns ?

Reply:
Yes.  As a best practice, we do everything possible to not run pods as a privileged user.

When the script is run from the default directory, the script tires to create files in a directory that it doesn't have permission to write.

With this change to /tmp, any user can run the script successfully.